api/app.py

```diff
@@ -3,8 +3,8 @@
 app = Flask(__name__)
 
 @app.route("/")
```

Comment:
Are we gonna end up protecting these endpoints with OAuth roles/permissions or something similar? Or just YOLO b/c Hackathon?

Reply:
fucking straight YOLO fam

Reply:
Noice!